tests/functional/b/bad_option_value.txt

```diff
@@ -1 +1,9 @@
-bad-option-value:1:0:None:None::Bad option value for enable. Don't recognize message W04044.:UNDEFINED
+bad-option-value:4:0:None:None::Bad option value for disable. Don't recognize message C05048.:UNDEFINED
```

Comment:
Do you mind doing a follow up PR just before the other PR which changes the confidence for `bad-option-value`?

Please don't do it in this PR as it already quite large.

Reply:
Yeah sure, I already have a lot of cherry-pick to do for 2.14.1 anyway 😄

Reply:
😅 

All good fixes though 😄